08b_Interactive_Catalog_Visualization.ipynb

```diff
@@ -242,9 +243,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "assert data[data[\"truth_type\"] == \"star\"].shape[0] == 2164\n",
-    "assert data[data[\"truth_type\"] == \"galaxy\"].shape[0] == 99932\n",
-    "assert data[data[\"truth_type\"] == \"SNe\"].shape[0] == 0"
+    "# assert data[data[\"truth_type\"] == \"star\"].shape[0] == 226 # 2164  \n",
+    "# assert data[data[\"truth_type\"] == \"galaxy\"].shape[0] == 10058 # 99932\n",
+    "# assert data[data[\"truth_type\"] == \"SNe\"].shape[0] == 0"
```

Comment:
Because if some of the quality criteria are removed from the query, there are SNe in the dataset. I want to alert the users to the fact that there are 3 truth types in the dataset,  and I want the plots to automatically include any SNe if the query is tweaked.

Reply:
Fair enough